src/utils/download.ts

```diff
@@ -1,3 +1,11 @@
+export const getBackupFileName = (): string => {
+    return `boltz-backup-${Math.floor(Date.now() / 1000)}-SECRET_DO_NOT_SHARE`;
+};
+
+export const getRefundFileName = (swap_id: string): string => {
+    return `boltz-refund-${swap_id}-SECRET_DO_NOT_SHARE`;
```

Comment:
```suggestion
export const getRefundFileName = (swapId: string): string => {
    return `boltz-refund-${swapId}-SECRET_DO_NOT_SHARE`;
```

We are not using snake case